.azure/pipelines/jobs/steps/dotnet-install.yml

```diff
@@ -1,4 +1,10 @@
 steps:
+  - task: UseDotNet@2
```

Comment:
I think these two lines ([1](https://github.com/sleepwalkerffs/trx2junit/blob/dotnet-6-update/.azure/pipelines/jobs/build_and_test.yml#L14), [2](https://github.com/sleepwalkerffs/trx2junit/blob/dotnet-6-update/.azure/pipelines/jobs/e2e_tests.yml#L8)) should be uncommented to make it work